packages/@azure/servicebus/data-plane/lib/session/messageSession.ts

```diff
@@ -523,7 +532,10 @@ export class MessageSession extends LinkEntity {
     }
     if (!options) options = {};
     this._isReceivingMessages = true;
-    this.maxConcurrentCallsPerSession = 1;
+    this.maxConcurrentCallsPerSession = 
+      (options.maxConcurrentCallsPerSession != undefined 
+        && typeof options.maxConcurrentCallsPerSession == "number"
```

Comment:
We can skip `options.maxConcurrentCallsPerSession != undefined` as we have `typeof options.maxConcurrentCallsPerSession == "number"`

Reply:
done